annotation-processor/src/main/kotlin/pl/touk/krush/source/MappingsGenerator.kt

```diff
@@ -1,28 +1,12 @@
 package pl.touk.krush.source
 
-import com.squareup.kotlinpoet.ClassName
-import com.squareup.kotlinpoet.FileSpec
-import com.squareup.kotlinpoet.FunSpec
-import com.squareup.kotlinpoet.ParameterSpec
+import com.squareup.kotlinpoet.*
 import com.squareup.kotlinpoet.ParameterizedTypeName.Companion.parameterizedBy
-import com.squareup.kotlinpoet.asClassName
-import com.squareup.kotlinpoet.asTypeName
 import org.jetbrains.exposed.sql.ResultRow
 import org.jetbrains.exposed.sql.statements.UpdateBuilder
 import pl.touk.krush.env.TypeEnvironment
-import pl.touk.krush.model.AssociationDefinition
-import pl.touk.krush.model.AssociationType.MANY_TO_MANY
-import pl.touk.krush.model.AssociationType.MANY_TO_ONE
-import pl.touk.krush.model.AssociationType.ONE_TO_MANY
-import pl.touk.krush.model.AssociationType.ONE_TO_ONE
-import pl.touk.krush.model.EntityDefinition
-import pl.touk.krush.model.EntityGraph
-import pl.touk.krush.model.EntityGraphs
-import pl.touk.krush.model.allAssociations
-import pl.touk.krush.model.asObject
-import pl.touk.krush.model.asVariable
-import pl.touk.krush.model.packageName
-import pl.touk.krush.model.traverse
+import pl.touk.krush.model.*
+import pl.touk.krush.model.AssociationType.*
```

Comment:
\[detekt\] INFO: \[style/WildcardImport\] Wildcard imports should be replaced with imports using fully qualified class names\. Wildcard imports can lead to naming conflicts\. A library update can introduce naming clashes with your classes which results in compilation errors\.